lib/ITip/Broker.php

```diff
@@ -345,7 +345,7 @@ protected function processMessageReply(Message $itipMessage, VCalendar $existing
 
         // Finding all the instances the attendee replied to.
         foreach($itipMessage->message->VEVENT as $vevent) {
-            $recurId = isset($vevent->{'RECURRENCE-ID'})?$vevent->{'RECURRENCE-ID'}->getValue():'master';
+            $recurId = isset($vevent->{'RECURRENCE-ID'}) ? $vevent->{'RECURRENCE-ID'}->getDateTime()->getTimestamp() : 'master';
```

Comment:
I'm curious why getTimestamp() was used instead of getValue(). The 'value' should always refer to the value in the timezone of the original event, so I don't believe there's a risk for timezone-related issues here.